frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts

```diff
@@ -115,6 +134,15 @@ if (clusterStatus && cephHealth) {
       expansionObjects.updatedPods = JSON.parse(
         execSync(`kubectl get pod -o json -n ${NS}`).toString(),
       );
+
+      expansionObjects.formattedOSDTree = createOSDTreeMap(
+        expansionObjects.osdTree.nodes,
+      ) as FormattedOsdTreeType;
+      expansionObjects.previousOSDIds = getIds(expansionObjects.osdTree.nodes, OSD);
+      expansionObjects.newOSDIds = getNewOSDIds(
```

Comment:
No, it will not. If you see closely, `expansionObjects.previousOSDIds,` is checked for its presence inside `expansionObjects.osdTree.nodes` as this contains all the osds and if not matched, then only it gets returned.

Reply:
That is right, but I see that when no new osds map is being given then it will always return [], because no new osds will be there.
`ceph osd tree` output is different after and before expansion.
My understanding says: 
1. osdTree = `ceph osd tree`
2. previousosdID <- osdTree
3. Expansion triggered
4. Then newosdTree = `ceph osd tree`
5. newosdId <- newosdTree, previousosdId

I ran the tests too , its showing empty array.
Please correct me if I am wrong.